index.html

```diff
@@ -95,16 +96,44 @@
+                  <path id="earphones" d="M278,314.79s4.81,44.64-35.56,44.8-33.86-44.8-33.86-44.8" transform="translate(-56.67 -185.6)" style="fill: none;stroke: #fff;stroke-linecap: round;stroke-miterlimit: 10;stroke-width: 4px"/>
+                </g>
+                <g id="rest">
+                  <path id="body" d="M389.24,435c-30.83-63.54-77.64-72.2-77.64-72.2V362H184.14s-44.9,7-78.7,72.09H189.7Z" transform="translate(-56.67 -185.6)" style="fill: #36476f;stroke: #0e1a32;stroke-miterlimit: 10;stroke-width: 4px"/>
+                  <rect x="122.21" y="231.3" width="131.76" height="14.6" style="fill: #fff;stroke: #000;stroke-miterlimit: 10;stroke-width: 4px"/>
+                  <path d="M310.69,426.63H179.59c-2.47,0-4.5-2.32-4.5-5.16V332.75H315.18v88.72C315.18,424.31,313.16,426.63,310.69,426.63Z" transform="translate(-56.67 -185.6)" style="fill: #565c68;stroke: #000;stroke-miterlimit: 10;stroke-width: 4px"/>
+                  <path d="M316,431.5a2.36,2.36,0,0,1-2.27,2.43H177.05a2.36,2.36,0,0,1-2.27-2.43h0c0-1.35,1-2.44,2-4.87H313.44c1.49,2.43,2.51,3.52,2.51,4.87Z" transform="translate(-56.67 -185.6)" style="fill: #606c71;stroke: #000;stroke-miterlimit: 10;stroke-width: 4px"/>
+                  <path d="M293.72,335s16.61,49.37-10.26,57-45.19,11.19-54.86,10.18-21.89-10.18-29.52-16.8-19.53-16.61-20.2-23.32,4.71-24.85,22.82-27.07S293.72,335,293.72,335Z" transform="translate(-56.67 -185.6)" style="fill: #373e46"/>
+                  <ellipse cx="187.01" cy="197.24" rx="6.27" ry="6.71" style="fill: #989fa4"/>
+                </g>
+                <line id="left" x1="118.43" y1="232.25" x2="73.23" y2="224.9" style="fill: #fff;stroke: #0d1932;stroke-miterlimit: 10;stroke-width: 4px"/>
+                <line id="right" x1="257.88" y1="232.25" x2="303.07" y2="224.9" style="fill: #fff;stroke: #0d1932;stroke-miterlimit: 10;stroke-width: 4px"/>
+                <line y1="249.4" x2="374.02" y2="249.4" style="fill: none;stroke: #0e1a32;stroke-miterlimit: 10;stroke-width: 4px"/>
+              </svg>
             <p class="about-content">
-                “Algorithms are not arbiters of objective truth and fairness simply because they're math.” - <i><b>Zoe Quinn</b></i>
-
+              
+               “Algorithms are not arbiters of objective truth and fairness simply because they're math.-”<i><b>Zoe Quinn</b></i></div>
+          
                 <br> <br>
-                <p class="text-justify">
-                    We are an organization of people who focuses on Data Structure and Algorithms. It is our goal to provide access to well written, well-explained programs and knowledge that improve people's lives. We want to enable people to adapt and change code according
-                    to their own ideas through an Open Access approach. We bring amazing people together to code algorithms and share our work for the benefit of all.
+                <div class="text-justify">
```

Comment:
use the `<p class="text-justify">` only or else correct the spacing if you plan to use the `div`.

Reply:
I have added the p class text justify and removed the div